example/extensions/lib_subgraph/README.md

```diff
@@ -107,15 +107,15 @@ The `optimize_for` API takes at least 1 argument, `backend` which is a string th
 For the Gluon API, `hybridize` can be called on HybridBlocks to partition the internal CachedOp Symbol.
 
 ```python
-block.hybridize(backend=None, backend_opts=None, clear=True, **kwargs)
+block.hybridize(backend=None, clear=True, **kwargs)
```

Comment:
should we change this now that the default for clear is False?

Reply:
Most of hybridize calls assume that `clear=True`, so I thought that it would be safer to keep it as it is.

Reply:
It's slightly different that the `optimize_for` case, the user might call several `optimize_for`, but will usually only call hybridize once at the beginning.